_posts/2017-10-27-QtAuto-on-Raspberry-Pi.md

```diff
@@ -0,0 +1,70 @@
+
+---
+layout: post
+title:  "Getting PELUX+QtAuto on the Raspberry Pi 3"
+date:   2017-10-27 15:00:00 +0200
+categories: yocto
+author: Tobias Olausson
+---
+
+For a while now, we have tried to get PELUX with the [Qt Automotive
+Suite](https://www1.qt.io/qt-automotive-suite/) onto a Raspberry Pi 3, to be
+able to show the same flavours of PELUX on both the Intel NUC and the Pi. After
+some hacking - it turns out it wasn't all that hard.
+
+## Starting out
+First, we just set `MACHINE` to "raspberrypi3" and built our PELUX image in
+yocto. We had one issue in that the graphics stack on the pi doesn't support GBM
+(when we use the broadcom proprietary drivers), so it had to be disabled when
+building Qt. After that - everything built super fine, the pi booted fine, but
+QtApplicationManager would just hang and we got no graphics output.
+
+Our Munich office contains the maintainers for
```

Comment:
Pretty long sentence there. Maybe it could be split up but I don't really have a strong opinion about it.